### DISCUSSION TITLE
Why "z" for redshift, and d = zc/H in particular should be rejected in a non-expanding universe

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#188] [Redshift Models]
**Dec 31, 2023**

I made a video called "Fixing redshift" on this topic. Feedback appreciated!

https://www.youtube.com/watch?v=AC2R4To96N8

Original:

----

In an expanding universe, three of the distances involved are pictured here:

![](https://raw.githubusercontent.com/mikehelland/hubbles-law/master/img/cosmodistances.png)

* distance at $t_{emit}$, angular diameter distance
* light travel time distance, $ct$
* distance at $t_{now}$, comoving distance

An exponentially expanding universe has a constant expansion rate, and has exact solutions for these distances. Respectively, they are:

$$d_A = \frac{c}{H_0} \frac{z}{1+z}$$

$$d_{ltt} = \frac{c}{H_0} \log (1+z)$$

$$d_C = z \frac{c}{H_0}$$

In LCDM, and indeed any FLRW model with a positive matter density, these equations predict distances that are too high. They are reigned in by matter in other models.

The greatest distance is obviously $d_C$. It's the distance the light was emitted from times (1+z). The light hasn't even traveled that far. 

It seems to me if the universe wasn't expanding, there would be no conceivable reason that distances would be the enormous distances that zc/H predicts, that are much larger than the distances the light traveled from, larger than the pre-expanded distances obviously, and even too high for LCDM.

Does that mean redshift and distance can't be linear?

No.

----

Here are four ways to quantify redshift, where $\lambda$ is a wavelength:

$$a = \frac{\lambda_{emit}}{\lambda_{obs}}$$

$$1 + b = \frac{\lambda_{emit}}{\lambda_{obs}}$$

$$1 - r = \frac{\lambda_{emit}}{\lambda_{obs}}$$

$$1 + z = \frac{\lambda_{obs}}{\lambda_{emit}}$$

These are all fundamentally the same. The only real difference is the numeric range redshfits fall in for each.

$$0 < a < 1$$

$$-1 < b < 0$$

$$0 < r < 1$$

$$0 < z < \infty$$

Outside of these ranges, for example when $z < 0$ or $b > 0$ is blueshift.

Any of these are valid choices, to be used as the value we record when we measure redshift, or in our equations.

z is kind of nice because things are spread out instead of crammed between 0 and 1 or -1. Notice it is only one with the ratio inverted. It sort of has this "distances are expanding vibe" built in. 

But z doesn't relate to distance very well, unlike the others. For the most part, in a big bang universe, z=1 is around half way back to the beginning. So half is between 0 and 1, the other half between 1 and infinity.

Notice you never see $z$ alone. It has one exact use, as the comoving distance in the exponentially expanding model $d = cz/H$, otherwise it is a mere low redshift approximation.

Everywhere else it appears is as $1+z$ or its inverse.

For this reason, $a$ is pretty useful in the equations, and is more or less the de factor way to reason about redshifts.

From 

**ΛCDM Cosmology for Astronomers**

[url]https://arxiv.org/pdf/1804.10047.pdf[/url]

>Because 1 ≤ (1 + z) < ∞ is the reciprocal of the scale
>factor 0 < a ≤ 1, at high redshifts both z and (1 + z)
>are very nonlinear and potentially misleading functions
>of fundamental quantities such as lookback time (Sec-
>tion 4.1). Had astronomers always been able to measure
>accurate frequency ratios νo/νe = a instead of just small
>differential wavelengths (λo −λe)/λe = z, most cosmolog-
>ical equations and results would probably be presented
>in terms of a today.

I think $1-r$ is a very interesting option. Using this, the exponentially expanding universe looks like this:

$$ 1 - r = \frac{1}{1+z}$$

$$  r = \frac{z}{z+z}$$

$$d_A = \frac{c}{H_0} r$$

It's between 0 and 1 for redshifts. The $r$ stands for redshift, but we also use $r$ for radius.

But if the radius $r$ is normalized to the Hubble length, say we give it a "natural cosmological unit" where $r = 1 = c/H$, then in an exponentially expanding universe (and therefore in my model too) the redshift $r$ = radius $r$.

In this case it's not just a linear relationship, it's straight up equal.

----

**Conclusion**

Using $z$ to quantify redshift seems like you're already admitting distances increase (expanding universe), and $zc/H$ is using the expanded distances in an exponentially expanding universe, which are actually themselves too high for LCDM.

The other quantifications of redshift lean more toward the losing energy and frequency interpretation, since they invert the traditional $z$ quantification.

Inverting $1+z$ actually reveals the true linear relationship between redshift and distance.

Is there any good reason to keep with the $z$ and $zc/H$ conventions? Especially in a nonexpanding paradigm?

**Comment by RedshiftDrift** [comment]
@mikehelland 
> Notice you never see $z$ alone.

Wrong, $z$ is directly derived from spectroscopic measurements, see image below.  It's actually the only model-independent quantity one sees "alone" in cosmology.  The only assumption made to calculate $z$ is that the wavelength $\lambda_{emit}$ at the time and location of emission was the same as the wavelength observed in the laboratory now.  (This assumption is strongly supported by our ability to recognize the spectra of hydrogen and every other element.)

Using $z$ does not imply any model for distance, the type of redshift mechanism, a variable speed of light nor a variable time scale, etc.  This is the only measured quantity in astronomy that transcends any paradigm because it is calculated directly from data.

However, I agree that
> $d = zc/H$ should be rejected in a non-expanding universe

(but I challenge you to convince [Eric Lerner](https://arxiv.org/abs/astro-ph/0509611) to reject this relationship).
All other physical quantities derived from $z$, such as $\displaystyle d = \frac{c}{H}z$, $\displaystyle r = \frac{c}{H} \frac{z}{1+z}$, $\displaystyle d = \frac{c}{H_0}\ln(1+z)$, $\displaystyle D = \frac{c}{H_0} \int_0^z \frac{{\rm d}z'}{\sqrt{\Omega_m(1+z')^3 + \Omega_\Lambda}}$, etc. only make sense in a specific paradigm and for the specific model they were derived from.

Note that these relationships involve the Hubble parameter $H_0$ which implies 'expansion' or 'recession' when given in units of *[km/s]/Mpc*.  The use of these units is what should be rejected in a non-expanding universe.  Instead, it is [preferable to use $H_z = 0.25$/*Gpc*](https://cosmology.info/code/marmet_l/cosmology-calculators.html) with units that imply an increasing redshift $\delta z$ per differential unit distance $\delta d$ as is observed for nearby galaxies, giving the differential equation

$\delta z = H_z \ \delta d  \quad  [1]$

How one integrates this equation depends on the underlying paradigm and how physical quantities like time, speed of light, distance, etc. depend on each other, since in general $z$, $H_z$ and $d$ can depend on $t$, $c$, $d$, $\rho$, $\alpha$, etc...

P.S. It is not clear to me how you can get $\displaystyle r = \frac{c}{H} \frac{z}{1+z}$ by integrating Eq. [1].  Han [clearly states the relationships between time and distance variables](http://www.alternatievewiskunde.nl/cosmology/Flandern.htm) to get $D = \frac{c_0}{H_i}\ln(1+z_i)$.  [My paper §9](http://arxiv.org/abs/1801.07582) also states how to integrate Eq. [1] $-$ while your equation seems to be pulled out of a hat.

![figure1](https://github.com/a-cosmology-group/acg/assets/16748402/6913ba6e-0173-469a-ad5c-02d439143dd6)

**Reply by HanDeBruijn** [reply]
@RedshiftDrift writes:
>while your equation seems to be pulled out of a hat.

Maybe not. It has cost me some search, but ..

Here is a series of slides, simply titled [Cosmology Slides](https://www.astro.rug.nl/~saleem/courses/CosmologySlides.pdf).
I think it's a decent overview of the current state in cosmological science(fiction).
Talking about _distances_ on slide 60:
>As mentioned earlier in the course within the frame of work of Robertson-Walker
metric one can define a number of distances. This is due to the fact that distances
are measured between objects that have different cosmic times, different
expansion rates (along or across the line of sight) etc. It is this mixing of time and
proper coordinates in the metric that gives rise to very different possibilities of
defining distances in the Universe.

On slide 47, indeed, two formulas as mentioned by @mikehelland are discovered:

![kopij_2](https://github.com/a-cosmology-group/acg/assets/135433222/7f1db992-478a-4ce4-851c-9dbc7f483428)

**Reply by mikehelland** [reply]
It's based on the speed of light being $v = c - Hd$.

I actually found that you can use $v = c - Hd$ to fully calculate an FLRW model. You can check out this notebook.

https://github.com/mikehelland/hubbles-law/blob/master/other/python/cosmodistance.ipynb

This assumes the - Hd comes from offsetting expanding space $v = Hd$.

In my model, this comes from time running slower. Time is observed to run slower at $z > 0$. Clocks run at $1/(1+z)$, or $a$.

The logic is:

* Light is observed at $c$ for all observers, on their local contemporary clock
* An observer in the past has a clock that runs at $1/(1+z)$ compared to ours
* They still measure light at $c$, on their clock
* Therefore, the light around them is moving at $v = c/(1+z)$ compared to our clock

If you take:

$$1 + z = \frac{\lambda_{obs}}{\lambda_{emit}}$$

Multiply both sides by $f_{obs} / f_{obs}$ you get:

$$1 + z = \frac{c}{v}$$

Where we don't know what $v$ is, but if $z > 0$ then $c > v$. Since $v < c$, then we can say $v = c - Hd$, where we don't know what $d$ it, but we know when we multiply it by $H$ we get a velocity, and if we subtract that from $c$, we get another velocity, that if we divide $c$ by it, well, we might get a $z$. 

$$c - Hd = \frac{c}{1+z}$$

$$\frac{1}{1+ z} = \frac{c-Hd}{c }$$

$$\frac{1}{1+ z} = 1 - \frac{Hd}{c }$$

$$\frac{z}{1+ z} = \frac{Hd}{c }$$

$$d = \frac{z}{1+ z} \frac{c}{H }$$

Well, that's a lovely thought and all, but does it work? This code verifies the results:

    data = [0]

    H = 70
    H = H / 3.08e19 * 60 * 60 * 24 * 365 * 1e6
    c = 1
    z = 0
    t = 0
    x = 0

    while x < 7000:
        t -= 1
        x += c / (1 + z)
        z = c / (c - H * x) - 1

Plot x over z.

**Reply by mikehelland** [reply]
Then plot t over z. You'll see the log() function.

>(but I challenge you to convince [Eric Lerner](https://arxiv.org/abs/astro-ph/0509611) to reject this relationship).

If someone is in touch with him, send this page his way.

**Reply by HanDeBruijn** [reply]
@mikehelland 

>Time is observed to run slower at $z\gt 0$

But then you have $1/(1+z)\lt 1$ so _Time is observed to run_ **faster** instead of slower.
This already destroys the rest of your reasoning, unless I've made an obvious mistake.

**Reply by RedshiftDrift** [reply]
Angular size measurements of galaxies show that we see them at a distance as far as 35 Gly that is much larger than a Hubble distance, yet $\displaystyle d = \frac{c}{H} \frac{z}{1+z}$ implies a maximum distance beyond which we should not see galaxies.

**Reply by mikehelland** [reply]
@RedshiftDrift 

> Angular size measurements of galaxies show that we see them at a distance as far as 35 Gly that is much larger than a Hubble distance, yet d=cHz1+z implies a maximum distance beyond which we should not see galaxies.

Is there a good dataset you recommend you to investigate that further?

Let's suppose $z=9$. LCDM gives a comoving distance of 30 Gly. 

My model says the galaxy is at a distance of $0.9 c/H$.

Accounting for the time dilation experienced by a photon traveling from $z=9$ to $z=0$,  is $c/H \log (1+9) = 32$ Gly.

The angular size the galaxy appears as should be based on this value. 

Note that at $z>8$ galaxies in my model have a larger luminosity distance than LCDM. 

@HanDeBruijn 

>But then you have $1/(1+z) < 1$ so Time is observed to run faster instead of slower.

Clock rates are $1/(1+z)$, or $a$.

So at $z=1$, the clock (there and then) is running at $1/2$ speed as compared to our clock .

**Reply by HanDeBruijn** [reply]
@mikehelland .
>Clock rates are $1/(1+z)$ or $a$

I don't know what you mean by "rate". Clock **periods** anyway are proportional to $(1+z)$.

So at $z=1$, the clock (there and then) is running with period $2$, that is indeed $1/2$ speed as compared to our clock .

But how about your purported derivation?

**Reply by RedshiftDrift** [reply]
@mikehelland A recent data set, which includes hundreds of galaxies and JWST observations, is given here [https://www.mdpi.com/2075-4434/10/6/108](https://www.mdpi.com/2075-4434/10/6/108).

**From Fig. 5** (copied below) with $d_A \propto 1/\theta$, comparing the angular size at $z_1=0.1$ and $z_2=10$ gives an observed ratio $\theta_1/\theta_2 = d_A(z_2)/d_A(z_1)$ of about $25$ (marked on the graph).

  Comparing that number with theory:
$\displaystyle d_A = \frac{c}{H}z \quad$ gives a ratio of $100$; the linear "*distance vs z*" model proposed by [Lerner](https://arxiv.org/abs/astro-ph/0509611) is falsified.
$\displaystyle d_A = \frac{c}{H} \frac{z}{1+z} \quad$ gives a ratio of $10.0$; your model @mikehelland is falsified.
$\displaystyle d_A = \frac{c}{H_0}\ln(1+z)$ (labeled 'TL' on the figure) gives a ratio of $25.2$ which fits the data very well.
$\displaystyle d_A = \frac{c}{H_0} \int_0^z \frac{{\rm d}z'}{\sqrt{\Omega_m(1+z')^3 + \Omega_\Lambda}} \quad$ gives a ratio of $2.29$; ***ΛCDM is very falsified***.

In the 'non-expansion paradigm', there is *no currently observed limit to the size of the universe* up to a distance of 35 Gly (z = 13.2; spectroscopically confirmed) and up to 40 Gly (z = 20; photometrically confirmed).

![angular-size](https://github.com/a-cosmology-group/acg/assets/16748402/a8fef89a-b7f7-4f83-9bc8-63528550609e)

**Reply by mikehelland** [reply]
@RedshiftDrift

Thanks for the link.

As mentioned in my previous comment, the physical distance ($(c/H) (z/(1+z))$) is not how it appears on the sky in my model. That's affected by time dilation.

"Accounting for the time dilation experienced by a photon traveling from $z=9$ to $z=0$,  is $c/H \log (1+9) = 32$ Gly. The angular size the galaxy appears as should be based on this value. "

**Reply by HanDeBruijn** [reply]
@mikehelland .
According to that scheme on slide 47 in [a previous reply](https://github.com/orgs/a-cosmology-group/discussions/188#discussioncomment-7645966) there would have been an opportunity to test your formula $d = z/(1+z).c/H$ in a universe with _Radiation alone_. But unfortunately that [Photon epoch](https://en.wikipedia.org/wiki/Chronology_of_the_universe#Photon_epoch) has "occurred" _Between 10 seconds and 370,000 years after the Big Bang_, which is quite a while ago :-(

**Reply by HanDeBruijn** [reply]
@RedshiftDrift .
A more or less preliminary version of this posting is found in [thread 160](https://github.com/orgs/a-cosmology-group/discussions/160#discussioncomment-7199278).
The mathematics below is meant as a heuristics for the 'TL' formula $\displaystyle d_A = \frac{c}{H_0}\ln(1+z)$ where we both agree upon.
![mike1](https://github.com/a-cosmology-group/acg/assets/135433222/4d7d239c-6c17-478d-9e49-b7aad0b512b5)
Take a look at the picture, depicting a timeline in cosmos back in time. Let (2) be our position on earth, (1) is the position of a far away civilization like ours, (0) is the position of a galaxy observed by observers on both habitable planets. Let $\lambda=$ wavelength of light. Then it is obvious that:

$$
\frac{\lambda_1}{\lambda_0} \frac{\lambda_2}{\lambda_1} = \frac{\lambda_2}{\lambda_0}
$$

And therefore, by definition of $z_{ab}=(\lambda_b-\lambda_a)/\lambda_a$ :

$$
1+z_{02} = (1+z_{01})(1+z_{12})
$$

It is reasonable to assume that the whole $\Delta t_{02}$ is the sum of the parts $\Delta t_{01}$ and $\Delta t_{12}$ and so

$$
1+z = F(\Delta t) \quad \mbox{where} \quad F(\Delta t_{02}) = F(\Delta t_{01}).F(\Delta t_{12}) = F(\Delta t_{01} + \Delta t_{12})
$$

This makes the function $F$ a special one, because it is seen now that the following property must hold for all $(t,\tau)$.

$$
\ln(F(t+\tau)) = \ln(F(t)) + \ln(F(\tau))
$$

We need another preliminary: [Additive functions and measure theory](https://math.stackexchange.com/questions/1867464/additive-functions-and-measure-theory). From that reference we infer that

$$
\ln(F(t)) = \kappa.t \quad \Longrightarrow \quad F(t) = 1+z = e^{\kappa.t}
$$

where $\kappa$ is a constant that has to be determined from physics. 
Employing $1+z = e^{\kappa.t} \approx 1+\kappa.t$ and Hubble's law $z = - H_0.t = \kappa.t$ for (relatively) small cosmic lightyears we get that $\kappa=-H_0$ .
And $d_A = -c.t$ then leads to the conclusion:

$$
1+z = e^{-H_0.t} \quad \Longrightarrow \quad d_A = \frac{c}{H_0}\ln(1+z)
$$

**Reply by mikehelland** [reply]
@HanDeBruijn 

The equation:

$$d = \frac{z}{1+z} \frac{c}{H_0}$$

appears as the angular diameter distance in de Sitter, $\Omega_\Lambda=1, \Omega_m=0$, and as the light travel time ins the Milne Model, $\Omega_\Lambda=0, \Omega_m=0$.

This is mostly a result of how the Friedmann equation works. Just because they have the same equation, doesn't mean they're the same. Here they represent two different things.

Your example from a "radiation dominated" universe is like it. Sure, it's the same equation. It means something different in that context though.

My model, unlike FLRW, doesn't have radiation, matter, and dark energy epochs. That's a different model entirely.

**Comment by mikehelland** [comment]
@RedshiftDrift 

> > Notice you never see z alone.
> 
> Wrong, z is directly derived from spectroscopic measurements, see image below. It's actually the only model-independent quantity one sees "alone" in cosmology. The only assumption made to calculate z is that the wavelength λemit at the time and location of emission was the same as the wavelength observed in the laboratory now. (This assumption is strongly supported by our ability to recognize the spectra of hydrogen and every other element.)
> 
> Using z does not imply any model for distance, the type of redshift mechanism, a variable speed of light nor a variable time scale, etc. This is the only measured quantity in astronomy that transcends any paradigm because it is calculated directly from data.

I understand your point. Correct me if I'm wrong, but I think every thing you said about $z$ could also be said for $a$, where:

$$a = \frac{\lambda_{emit}}{\lambda_{obs}}$$

What I meant by "see it alone" wasn't very clear. I meant in equations you never see $z$ by itself (with the exception I gave of $d_C$ in a de Sitter universe). You see it as $1+z$, or even more often, $(1+z)^{-1}$.

For $z$ to be a positive number that grows as wavelength grows does have an implicit bias to redshift being "primarily" a distance phenomena, rather than an energy or frequency phenomena. Does that make any sense? It's inconsequential to the math of course. You can just substitute $a$ and call it day.  But maybe one more clearly depicts reality while the other obfuscates it? That's kind of what I'm getting at.
